docs/hld.md

```diff
@@ -0,0 +1,245 @@
+# Family Hubs High Level Design Document
+
+## Glossary
+
+- FH - Family hubs
+- FX - Family experience
+- IS - Information Sharing
+- MVS - Minimal Viable Service
+- VCS – Voluntary Community Services
+- LA – Local Authority
+
+## Problem Statement
+
+This document provides details of the high-level design for the Vulnerable Children and Families portfolio Family Hubs service directory and request for support products. It consists of three main components:
+
+1. **Find:** Allows public users to browse a national service directory
+2. **Connect:** Allows professional and voluntary users to refer families to services
+3. **Manage:** Allows administration of service data and user accounts
+
+## Architecture
+
+The following diagram provides a high-level overview of the Family Hubs architecture.
+
+**Note:** as this is only a high-level overview, it is not intended to be a 1:1 visualisation of the entire Family Hubs architecture.
+
+### High-level Architecture
+
+```mermaid
+
+graph LR
+    %%{init:{'flowchart':{'nodeSpacing': 50, 'rankSpacing': 100 }}}%%
+
+    style PublicUsers fill:#FFF1E6,stroke:#333
+    style ProfessionalUsers fill:#FFF1E6,stroke:#333
+    style AdministrativeUsers fill:#FFF1E6,stroke:#333
+    style FamilyHubsDev fill:#FFF1E6,stroke:#333
+
+    PublicUsers[Public Users] -->|Use| Find[Find]
+    ProfessionalUsers[Professional Users] -->|Authenticate using| GovOneLogin[Gov One Login] -->|Use| Connect[Connect]
+    ProfessionalUsers[Professional Users] -->|Authenticate using| GovOneLogin -->|Use| ConnectDashboard[Connect Dashboard]
+    AdministrativeUsers[Administrative Users] -->|Authenticate using| GovOneLogin -->|Use| Manage[Manage]    
+    FamilyHubsDev[Family Hubs Developers] -->|Use| IdAMMaintenanceUI[IdAM Maintenance]
+
+    style Find fill:#F0EFEB,stroke:#333
+    style Connect fill:#F0EFEB,stroke:#333
+    style ConnectDashboard fill:#F0EFEB,stroke:#333
+    style Manage fill:#F0EFEB,stroke:#333
+    style IdAMMaintenanceUI fill:#F0EFEB,stroke:#333
+
+    subgraph Azure
+        style Azure fill:#e0f7fa,stroke:#333,stroke-dasharray: 5, 5,margin:102px
+        
+        subgraph App_Services[App Services]
+            style ServiceDirectoryAPI fill:#DFE7FD,stroke:#333
+            style IdAMAPI fill:#DFE7FD,stroke:#333
+            style ReferralAPI fill:#DFE7FD,stroke:#333
+            style ReportAPI fill:#DFE7FD,stroke:#333
+            style NotificationAPI fill:#DFE7FD,stroke:#333
+            
+            Find -->|Connects to| ServiceDirectoryAPI[Service Directory API]
```

Comment:
optional - could 'UI' be added to the UI's on the diagram to make it explicit that they are (Gary also mentioned)

Reply:
Removed them but happy to add them back if it makes it clearer :)